src/colmap/exe/sfm.cc

```diff
@@ -183,6 +183,66 @@ int RunColorExtractor(int argc, char** argv) {
   return EXIT_SUCCESS;
 }
 
+int RunIncrementalModelRefiner(int argc, char** argv) {
```

Comment:
A lot of this is very similar to `RunPointTriangulator`. Would it make sense to instead augment `RunPointTriangulator` with this new feature? can we minimally allow the user the control whether to 1) fix all images (the default of `RunPointTriangulator`), 2) fix no image, 3) fix a subset of images?